operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1661,7 +1661,7 @@ metaMonitoring:
     # unless the Prometheus Operator is set up to look for rules outside its namespace
     namespace: null
     # -- Contents of Prometheus rules file
-    groups: []
+    groups: ['{{ .Files.Get "mimir-distributed/mixins/alerts.yaml" }}']
```

Comment:
This complains for me because the string isn't evaluated as a template. I think the approach with the flag that you have below will be easier for users